docs/specs/properties.md

```diff
@@ -15,9 +15,13 @@ Their structure depends on whether it's animated or not:
 
 {schema_object:properties/base-keyframe}
 
+Keyframe arrays MUST be stored in order of ascending frame number.
```

Comment:
We might want to say that they must be strictly ascending, two keyframes cannot have the same `t` value

Reply:
Updated